src/Modules/Workspaces/Commands.Workspaces/GetPowerBIWorkspace.cs

```diff
@@ -111,6 +108,12 @@ public override void ExecuteCmdlet()
                 return;
             }
 
+            if (this.All.IsPresent && this.Scope == PowerBIUserScope.Organization)
+            {
+                this.ExecuteCmdletWithAll();
+                return;
+            }
+
             if (this.Deleted.IsPresent)
             {
                 this.Filter = string.IsNullOrEmpty(this.Filter) ? DeletedFilterString : $"({this.Filter}) and ({DeletedFilterString})";
```

Comment:
do we still need this code?

Reply:
Without providing the -All parameter, this code will do the filtering on deleted workspaces by sending it to the backend. So we might still want to support -Deleted along with -First and -Skip right?